net-analyzer/sn0int/sn0int-0.12.0.ebuild

```diff
@@ -0,0 +1,453 @@
+rustc_version-0.2.3
+rusticata-macros-1.1.0
+rustls-0.15.2
+rustyline-4.1.0
+ryu-0.2.8
+safemem-0.3.0
+same-file-1.0.4
+schannel-0.1.15
+scheduled-thread-pool-0.2.1
+scoped-tls-0.1.2
+scoped_threadpool-0.1.9
+scopeguard-0.3.3
+scopeguard-1.0.0
+sct-0.5.0
+seccomp-sys-0.1.2
+security-framework-0.3.1
+security-framework-sys-0.3.1
+selectors-0.21.0
+semver-0.9.0
+semver-parser-0.7.0
+separator-0.4.1
+serde-1.0.92
+serde_derive-1.0.92
+serde_json-1.0.39
+serde_urlencoded-0.5.5
+servo_arc-0.1.1
+sha-1-0.8.1
+sha2-0.7.1
+sha2-0.8.0
+sha3-0.8.2
+shellwords-1.0.0
+siphasher-0.2.3
+slab-0.4.2
+sloppy-rfc4880-0.1.3
+smallvec-0.6.10
+sn0int-0.12.0
+sn0int-common-0.7.0
+sn0int-registry-0.5.0
+socket2-0.3.9
+spin-0.5.0
+stable_deref_trait-1.1.1
+state-0.4.1
+string-0.2.0
+string_cache-0.7.3
+string_cache_codegen-0.4.2
+string_cache_shared-0.3.0
+strsim-0.8.0
+structopt-0.2.16
+structopt-derive-0.2.16
+strum-0.15.0
+strum_macros-0.15.0
+subtle-1.0.0
+syn-0.11.11
+syn-0.15.36
+synom-0.11.3
+synstructure-0.10.2
+syscallz-0.11.4
+tar-0.4.25
+tempfile-3.0.8
+tendril-0.4.1
+termcolor-1.0.5
+termion-1.5.3
+textwrap-0.11.0
+thin-slice-0.1.1
+thread_local-0.3.6
+threadpool-1.7.1
+tiff-0.2.2
+time-0.1.42
+tokio-0.1.21
+tokio-buf-0.1.1
+tokio-codec-0.1.1
+tokio-core-0.1.17
+tokio-current-thread-0.1.6
+tokio-executor-0.1.7
+tokio-fs-0.1.6
+tokio-io-0.1.12
+tokio-reactor-0.1.9
+tokio-rustls-0.9.3
+tokio-sync-0.1.6
+tokio-tcp-0.1.3
+tokio-threadpool-0.1.14
+tokio-timer-0.2.11
+tokio-trace-core-0.2.0
+tokio-udp-0.1.3
+tokio-uds-0.2.5
+toml-0.4.10
+toml-0.5.1
+traitobject-0.1.0
+trust-dns-0.16.1
+trust-dns-proto-0.7.4
+try-lock-0.2.2
+try_from-0.3.2
+typeable-0.1.2
+typenum-1.10.0
+ucd-trie-0.1.1
+ucd-util-0.1.3
+unicase-1.4.2
+unicase-2.4.0
+unicode-bidi-0.3.4
+unicode-normalization-0.1.8
+unicode-segmentation-1.3.0
+unicode-width-0.1.5
+unicode-xid-0.0.4
+unicode-xid-0.1.0
+untrusted-0.6.2
+unveil-0.2.0
+url-1.7.2
+utf-8-0.7.5
+utf8-ranges-1.0.3
+utf8parse-0.1.1
+uuid-0.7.4
+vcpkg-0.2.6
+vec_map-0.8.1
+version_check-0.1.5
+void-1.0.2
+walkdir-2.2.8
+want-0.0.6
+webpki-0.19.1
+webpki-roots-0.16.0
+widestring-0.4.0
+winapi-0.2.8
+winapi-0.3.7
+winapi-build-0.1.1
+winapi-i686-pc-windows-gnu-0.4.0
+winapi-util-0.1.2
+winapi-x86_64-pc-windows-gnu-0.4.0
+wincolor-1.0.1
+winconsole-0.10.0
+winreg-0.6.0
+winutil-0.1.1
+ws2_32-sys-0.2.1
+x509-parser-0.4.2
+xattr-0.2.2
+xml-rs-0.8.0
+yansi-0.4.0
+yansi-0.5.0
+"
+
+inherit cargo bash-completion-r1 fcaps
+
+DESCRIPTION="Semi-automatic OSINT framework and package manager"
+HOMEPAGE="https://github.com/kpcyrd/sn0int"
+SRC_URI="$(cargo_crate_uris ${CRATES})"
+RESTRICT="mirror"
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+man"
+
+RDEPEND="sys-libs/libseccomp
+		dev-db/sqlite"
+DEPEND="${RDEPEND}
+		>=virtual/rust-1.34.2
+		man? ( dev-python/sphinx )"
+
+src_compile() {
+	cargo_src_compile
+
+	use man && make -C docs man
+
+	mkdir -p completions
+	"target/release/sn0int" completions bash > completions/sn0int.bash
```

Comment:
Looks like these use the already compiled product to create shell completions, smart! Still, they all need to be appended with `|| die`s as well.